.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1 @@
+Be sure you are following the instructions in the [dev guidelines](http://www.numpy.org/devdocs/dev/gitwash/development_workflow.html) especially include a properly formatted [commit message](http://www.numpy.org/devdocs/dev/gitwash/development_workflow.html#writing-the-commit-message).
```

Comment:
Maybe better to put this in `<!-- -->` so that users don't submit this text?

Reply:
By default, for a single commit, github will just use the commit message as initial text. This is probably worth mentioning, as it implies that with a good commit message you do not need a pull request template.

Reply:
I couldn't find a guide to the differences between text inside `<< >>` and `<!-- -->`, do you know why SciPy used `<< >>` ?

Reply:
So if it will not render, why would we want to use `<!-- -->`? It seems I don't understand the [review comment](https://github.com/numpy/numpy/pull/11512#discussion_r200475911) and how these templates are meant to work.

Reply:
Because when someone creates a new issue, they see it pre-rendered. Anything that's an instruction to them, but is useless to us, should be visible only to them.

What I'm trying to avoid here is people submitting the template without modifications, and us seeing a lot of junk. Obviously they shouldn't do that, but I'd rather the result is a blank issue than one that clutters our search results.

Reply:
ok, fixing